### ISSUE TITLE
The continuous rising of the second stage loss when training the model from scratch

### ISSUE THREAD

**Issue by YkiWu** [CUT3R/CUT3R#43]
Excellent work!
I want to train your model from scratch, and I have prepared several static datasets (Co3d, ARKitScenes, Scannetpp, BlendedMVS, Megadepth), hoping to train the model's ability to reconstruct static scenes.
I trained the first stage model (224 * 224, linear head) according to stage1.yaml you provided and the test set curve looks normal.

<img width="716" alt="Image" src="https://github.com/user-attachments/assets/a37d8a05-fc75-4491-9954-fe752b349838" />

Then I used this checkpoint to train the next stage model(512, dpt head) according to stage3.yaml you provided (as stage2 involves dynamic datasets), however, the test loss of the pts3d reconstruction seems to be slowly rising in fluctuations as shown in the following figure.

<img width="713" alt="Image" src="https://github.com/user-attachments/assets/9cf9da4b-2da1-4edb-8509-2bd384d9184f" />

All the changes I made to the config file were to keep only the datasets I prepared, modify the corresponding data paths, and appropriately reduce the number of sampled datasets in each epoch during training to save training time, as shown in the following figure.

MyStage1 (Your stage1)

<img width="1136" alt="Image" src="https://github.com/user-attachments/assets/751d23c2-ebdd-4059-9b65-c630399a9fe2" />

MyStage2 (Your stage3)

<img width="1128" alt="Image" src="https://github.com/user-attachments/assets/6a553fd4-4cd3-40f4-91e7-5301acc9cf42" />
 
According to your stage 3, the model training at this stage is also coming to an end (40 epochs). I would like to know if this loss change is normal? Or did I miss something important?
Looking forward to your reply!

**Comment by Forrest-110** [comment]
Hi @YkiWu ! Apologies for the late reply. This is not normal. Have you tried to train it longer to see if the loss eventually goes down? My guess is that the learning rate might need to be modified.
